fda/basis.py

```diff
@@ -276,6 +276,12 @@ def copy(self):
         """Basis copy"""
         return copy.deepcopy(self)
 
+    def tofdatabasis(self):
```

Comment:
It should be `to_basis` to be consistent with the FData API. I know that the name is confusing, and maybe it should be changed later, but it should be the same.

Reply:
Ok